internal/gatewayapi/testdata/grpcroute-with-valid-ratelimitfilter.in.yaml

```diff
@@ -0,0 +1,55 @@
+gateways:
```

Comment:
should we also add some samples in `example/kubernetes`? and also add some doc for this?